be/test/vec/function/function_test_util.h

```diff
@@ -352,4 +352,36 @@ Status check_function(const std::string& func_name, const InputTypeSet& input_ty
     return Status::OK();
 }
 
+using BaseInputTypeSet = std::vector<TypeIndex>;
+
```

Comment:
add some comment： what different with `arg_comb` and `check_function`

Reply:
If the function has two arguments, I want to test it, since each argument maybe const, So there are four combinations of parameters, check_function can only accept one combination at a time, If you want to test it all, you need to call it four times, while the function with arg_comb only needs to be called once